tools/build_config/Jenkinsfile

```diff
@@ -1,18 +1,22 @@
 #!groovy
 
 def post_github_status(String state, String message) {
-    withCredentials([string(credentialsId: 'GitHub_API_Token',
-            variable: 'api_token')]) {
-        sh """
-            curl -H "Authorization: token ${api_token}" \
-                --request POST \
-                --data '{"state": "${state}", \
-                         "description": "${message}", \
-                         "target_url": "$BUILD_URL", \
-                         "context": "$JOB_BASE_NAME"}' \
-                $PR_STATUSES_URL > /dev/null
-        """
-            }
+    // Non-PR builds will not set PR_STATUSES_URL - in which case we do not
```

Comment:
Builds of `<named-branch>` (master or anything else) should still set build status flags if at all possible. 

If the build fails we need to know!

Reply:
Created new ticket (#51) to resolve this issue; the simple extensions attempted (below) have failed.